src/build_vehicle_gui.cpp

```diff
@@ -1321,20 +1322,16 @@ struct BuildVehicleWindow : Window {
 		this->te.FillDefaultCapacities(e);
 	}
 
-	void OnInit() override
-	{
-		this->SetCargoFilterArray();
-	}
-
 	/** Filter the engine list against the currently selected cargo filter */
```

Comment:
Doxygen needs updating to show the return type.

Reply:
None of the functions around have a full doxygen comment so idk what the proper way of commenting here. But I added some mention of return value to the comment.